### ISSUE TITLE
load-with-code-conversion: Symbol’s value as variable is void: x

### ISSUE THREAD

**Issue by Whiothes** [syl20bnr/spacemacs#9396]
I got this message after running
```
git pull --rebase
```
Anyone can help me ?  THX!

#### Description :octocat:
load-with-code-conversion: Symbol’s value as variable is void: x

#### Reproduction guide :beetle:
1.  
    - Start Emacs
    - **Message: load-with-code-conversion: Symbol’s value as variable is void: x**
    - Stop Emacs
2. 
    - `rm -rf elpa`
    - `dotspacemacs-elpa-https nil`
    - Start Emacs
    - **Message: load-with-code-conversion: Symbol’s value as variable is void: x**
    - **Message: File mode specification error: (void-function flycheck-ycmd-setup)**

*Observed behaviour:* :eyes: :broken_heart:
- I have already tried: [FAQ](https://github.com/syl20bnr/spacemacs/blob/master/doc/FAQ.org#how-to-fix-symbols-value-as-variable-is-void-errors-on-startup ) 

*Expected behaviour:* :heart: :smile:
Running normally

#### System Info :computer:
- OS: darwin
- Emacs: 25.2.1
- Spacemacs: 0.200.9
- Spacemacs branch: develop (rev. 459268f5)
- Graphic display: t
- Distribution: spacemacs
- Editing style: emacs
- Completion: helm
- Layers:
```elisp
(csv plantuml sql
     (ycmd :variables ycmd-force-semantic-completion t ycmd-server-command
           (list "python"
                 (file-truename "~/Public/Projects/YouCompleteMe/third_party/ycmd/ycmd"))
           ycmd-global-config
           (file-truename "~/Templates/.ycm_extra_conf.py")
           ycmd-extra-conf-whitelist
           '("~/*"))
     chinese-fonts-setup
     (gtags :variables gtags-enable-by-default t)
     markdown html
     (python :variables python-enable-yapf-format-on-save t)
     syntax-checking
     (c-c++ :variables c-c++-default-mode-for-headers 'c++-mode c-c++-enable-clang-support t)
     colors imenu-list helm
     (auto-completion :variables auto-completion-return-key-behavior 'complete auto-completion-tab-key-behavior 'cycle auto-completion-complete-with-key-sequence nil auto-completion-complete-with-key-sequence-delay 0.1 auto-completion-private-snippets-directory nil auto-completion-enable-sort-by-usage t auto-completion-enable-help-tooltip t)
     better-defaults emacs-lisp
     (git :variables magit-repository-directories
          '("~/Private/Projects" "~/Public/Projects" "~/Documents/"))
     (org :variables org-enable-org-journal-support t org-journal-dir "~/Git/org/journal/" org-journal-file-format "%Y-%m-%d.org" org-journal-date-prefix "#+TITLE: " org-journal-date-format "%A, %B %d %Y" org-journal-time-prefix "* " org-journal-time-format "" org-projectile-per-project "org/TODOs.org" org-enable-github-support t org-capture-templates
          '(("n" "Note" entry
             (file "~/Git/org/NOTEs.org")
             "* %? %T\n" :prepend t :jump-to-captured t :empty-lines 1)
            ("t" "Todo" entry
             (file "~/Git/org/TODOs.org")
             "* TODO %?\n %t" :empty-lines 1 :jump-to-captured t)
            ("a" "Agenda" entry
             (file+datetree "~/Git/org/agenda.org")
             "" :empty-lines 1 :jump-to-captured t)
            ("z" "zxpay/CHANGELOG" entry
             (file+headline "~/Working/zxpay/org/CHANGELOG.org" "Release 0.x.x")
             "* %? %t\n** New\n** Fix" :prepend t :jump-to-captured t)))
     (markdown :variables markdown-live-preview-engine 'vmd)
     (ibuffer :variables ibuffer-group-buffers-by 'projects)
     (shell :variablesp shell-default-shell 'eshell shell-default-height 30 shell-default-position 'bottom shell-enable-smart-eshell t)
     (spell-checking :variables spell-checking-enable-by-default nil)
     (syntax-checking :variables syntax-checking-enable-by-default nil syntax-checking-enable-tooltips nil)
     (spacemacs-layouts :variables layouts-enable-autosave nil layouts-autosave-delay 300)
     version-control chrome osx)
```
- System configuration features: NOTIFY ACL GNUTLS LIBXML2 ZLIB TOOLKIT_SCROLL_BARS NS MODULES

**Comment by bmag** [comment]
Please run `emacs --debug-init` and post the backtrace.

**Comment by Whiothes** [comment]
I'm not sure what is important , so I post the whole message here
[message.txt](https://github.com/syl20bnr/spacemacs/files/1213829/message.txt)

**Comment by bmag** [comment]
That's not a backtrace, those are just messages. If you run the command `emacs --debug-init` and there's an error during startup, then a backtrace will be shown in the `*Backtrace*` buffer.

**Comment by Whiothes** [comment]
I use the command `/usr/local/bin/emacs --debug-init`, but I didn't get such buffer.

But I got this message after calling function `cnfonts-*` every time, but others not. 
it seems this message just caused because of the package `chinese-fonts-setup`. I removed this one , and it disappeared now.

So, it should be the upgrade of the package `chinese-fonts-setup` caused the problem, not *spacemacs*.

Thank you a lot , and I'll close the issue.
